mllib/src/main/scala/org/apache/spark/mllib/feature/Word2Vec.scala

```diff
@@ -431,6 +431,14 @@ class Word2Vec extends Serializable with Logging {
 class Word2VecModel private[mllib] (
     private val model: Map[String, Array[Float]]) extends Serializable with Saveable {
 
+  private val numDim = model.head._2.size
```

Comment:
Use "vectorSize" instead of "numDim" to mimic Word2Vec class.
Better yet, move this and flatVec into a constructor or closure which sets up wordVecMat, wordVec (since this and flatVec are not used anywhere else)